ibtk/src/utilities/HierarchyIntegrator.cpp

```diff
@@ -558,6 +558,11 @@ void
 HierarchyIntegrator::registerVisItDataWriter(Pointer<VisItDataWriter<NDIM> > visit_writer)
 {
     d_visit_writer = visit_writer;
+    if (d_workload_idx != IBTK::invalid_index)
```

Comment:
Let's include a run-time option to turn this on/off. Seems fine to me to default to "on".